server/repository/src/db_diesel/name.rs

```diff
@@ -45,6 +47,8 @@ pub struct NameFilter {
     pub address2: Option<StringFilter>,
     pub country: Option<StringFilter>,
     pub email: Option<StringFilter>,
+
+    pub code_or_name: Option<StringFilter>,
```

Comment:
I can sense the request coming in to have also the boolean filters for supplier/customer/donor.